group_vars/mainnet.yml

```diff
@@ -4,6 +4,8 @@ sui:
   validator:
     version: 615516edb0ed55e45d599f042f9570b493ce9643
     external_address: validator.sui.mainnet.encapsulate.xyz
+    ports:
```

Comment:
override the testnet file please, make the all.yml the standard one with the default ports

Reply:
add a comment too, overriding default ports to support haproxy